BoxContentSDK/BoxContentSDK/OAuth2/BOXOAuth2Session.h

```diff
@@ -156,6 +156,12 @@
  */
 - (NSString *)redirectURIString;
 
+/**
+ * Returns the randomly generated nonce used to prevent spoofing attack during login
+ * @return generated nonce
+ */
+- (NSString *)nonce;
```

Comment:
Any reason that we should not expose this in the header? (in other words, in your thought process what were the pros and cons, or does this just make sense as-is?)